webpack.config.js

```diff
@@ -0,0 +1,27 @@
+const slsw = require('serverless-webpack')
+const nodeExternals = require('webpack-node-externals')
+
+module.exports = {
+  entry: slsw.lib.entries,
+  stats: 'minimal',
+  target: 'node',
+  devtool: 'source-map',
+  externals: [nodeExternals()],
+  mode: slsw.lib.webpack.isLocal ? 'development' : 'production',
+  optimization: {
+    minimize: false,
```

Comment:
Couldn't be ingore to use default value related to `mode` ?

Reply:
Oh yeah!